tx-source/dash_core.php

```diff
@@ -8,6 +8,7 @@
 $Definition['%s item in queue'] = '%s item in queue';
 $Definition['%s level'] = '%s level';
 $Definition['%s levels'] = '%s levels';
+$Definition['%s prohibited from using the ignore feature.'] = '%s prohibited from using the ignore feature.';
```

Comment:
This doesn't seem grammatically correct.

Reply:
It creates a grammatically correct sentence when populated by the $ReferTo variable. Is this something you want changed?